polkadot/node/service/Cargo.toml

```diff
@@ -219,10 +223,7 @@ try-runtime = [
 	"sp-runtime/try-runtime",
 	"westend-runtime?/try-runtime",
 ]
```

Comment:
We do need to add `"xcm-builder/runtime-benchmarks"` here 🤦 
I don't know why that was failing on my machine

Reply:
Let me examine the dependencies. I think we added some unneeded ones.

Reply:
@mrshiposha

Reply:
We don't need the `xcm-builder` here at all. We use only the `xcm` to get the XCM types.

Reply:
Even better then